draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2592,38 +2596,43 @@ http://example.com/hls/35cdc008/audio/index.m3u8\n”,
 
 
       <section anchor="ccids-spec" title="CCIDs Spec">
-        <t> The content-ccids spec type allows the uCDN to specify the
+        <t> The "content-ccids" spec type allows the uCDN to specify the
         Content Collection IDentifier of content to which the trigger applies.
         The "ccid" is a grouping of content as defined by <xref target="RFC8006"/>.
         The ccid spec type is valid only for the content spec subject (see <xref target="trigger-subject"/>).
 
         </t>
+	<t>CCIDs spec is encoded as a JSON object containing the following name/value pairs:</t>
 
-         <t>Spec object specification<list style="empty">
-
-              <t>Property: ccids<list style="empty">
-
-                 <t>Description: An array of Content Collection IDentifiers over which the trigger MUST be executed.</t>
+        <t><list style="empty" hangIndent="3">
+        <t><list style="hanging" hangIndent="3"><t hangText="Name: ccids">
+        <vspace blankLines="1"/>
+	Description: An array of Content Collection IDentifiers over which the trigger MUST be executed.
+        <vspace blankLines="1"/>
+	Value: A JSON array of strings, where each string is a Content Collection IDentifier.
+        <vspace blankLines="1"/>
+        Mandatory: Yes.
+        </t>
 
-                 <t>Value: A JSON array of strings, where each string is a Content Collection IDentifier.</t>
+        </list></t>
+        </list></t>
 
-                 <t>Mandatory: Yes.</t>
-              </list></t>
-          </list></t>
       </section>
 
-     <section anchor="uri-pattern-match-spec" title="URI Pattern Match Spec">
-        <t>The uri-pattern match spec type allows the uCDN to manage uCDN content or metadata objects
-           held by the dCDN based on the objects' URI pattern. The value is a UriPatternMatch object, 
-		as defined in <xref target="url-pattern-match"/>.</t>
+     	<section anchor="uri-pattern-match-spec" title="URI Pattern Match Spec">
+	<t>
+		The "uri-pattern" match spec type allows the uCDN to manage uCDN content or metadata objects
```

Comment:
Shouldn't match be included in the quotes?  Or uri-pattern-match?

Reply:
There's broader v12 issue, the spec types defined in Section 3.2 were used rather liberally
Fixed